backend/infrahub/events/branch_action.py

```diff
@@ -17,9 +17,10 @@ class BranchDeletedEvent(InfrahubEvent):
 
     def get_resource(self) -> dict[str, str]:
         return {
-            "prefect.resource.id": f"infrahub.branch.{self.branch_name}",
-            "infrahub.branch.id": self.branch_id,
-            "infrahub.branch.name": self.branch_name,
+            "prefect.resource.id": f"infrahub.branch.{self.branch_id}",
+            "infrahub.node.kind": "Branch",
```

Comment:
not sure about this one
I would expect the kind to be part of the schema but Branch is not technically part of the schema :/ 
Every client will need to manage an exception just for that (edited)